java/src/jmri/jmrix/dccpp/DCCppInitializationManager.java

```diff
@@ -60,7 +60,7 @@ public DCCppInitializationManager(DCCppSystemConnectionMemo memo) {
             version         = cs.getVersion();
         }
         
-        log.info("DCC++ Initialization Complete with station type '{}', version '{}' and build '{}'", base_station, version, code_build);
+        log.info("DCC++ Initialization Complete");
```

Comment:
A simple solution to solve the "unused variable" is to have two rows:
```
        log.trace("DCC++ Initialization Complete with station type '{}', version '{}' and build '{}'", base_station, version, code_build);
        log.info("DCC++ Initialization Complete");
```
In normal case, only the log.info() will print to the log. But it's still possible to get all info to the log  by changing logging to "trace".

https://www.jmri.org/help/en/html/doc/Technical/Logging.shtml

Reply:
Thanks for the suggestion, but those values are never going to be set in time for that message. So I completed the cleanup of the unused variables.

Reply:
You don't seem to have commited the cleanup to GitHub.